lib/adapters/REST/endpoints/asset.ts

```diff
@@ -16,13 +16,16 @@ import { normalizeSelect } from './utils'
 
 export const get: RestEndpoint<'Asset', 'get'> = (
   http: AxiosInstance,
-  params: GetSpaceEnvironmentParams & { assetId: string } & QueryParams
+  params: GetSpaceEnvironmentParams & { assetId: string } & QueryParams,
+  _rawData?: never,
```

Comment:
Is it right to pad for the additional `rawData` parameter?

Reply:
I guess we can just skip rawData parameter and add headers as a third one

Reply:
Would it receive the correct parameter from [here](https://github.com/contentful/contentful-management.js/pull/808)?

Reply:
from where?

Reply:
@suevalov sorry 🤦  here https://github.com/contentful/contentful-management.js/blob/master/lib/adapters/REST/rest-adapter.ts#L84

Reply:
hmmm... it's very clunky :/ @andipaetzold is there anything we can with adapters API to avoid so confusing any of skipping one param?

Reply:
I come from verifying this. If we call `entry.get(params, headers)` then `headers`, the 2nd argument, ends up as the `payload` argument

Reply:
@IBratWurst ping @andipaetzold on Slack and ask his opinion here

Reply:
@suevalov he's on vacation, till April 25 according to bambooh. Good for him but not for our desired feedback 😕

Reply:
okay, let's go with `_rawData?: unknown` for now and merge the PR